### PR TITLE
[Dygraph] Tensor Parallelism improve & accuracy alignment

### PR DESCRIPTION
**Pull request by JZ-LIANG**
### PR types
Bug fixes

### PR changes
OPs

### Describe
1. bugfix: add "is_distributed" attribute to splited bias tensor
2. performance improvement: tensor split c_identity op support FP16 (add to gray list)
in the env: 4 x V100, Ernie-base, bsz=64, speed improve 11.6%

|  Ernie-base   | token/s |
|  ----  | ----  |
| mp4-before pr  | 28491 |
| mp4-after pr  | 31790 |

### CODE REVIEW

**Review by sandyhouse — DISMISSED** [review]
LGTM

File: python/paddle/distributed/collective.py [hunk 1]
```diff
@@ -1008,6 +1008,10 @@ def _parallel_linear(x,
     main_block = paddle.static.default_main_program().global_block()
     startup_block.vars[linear.weight.name].is_distributed = True
     main_block.vars[linear.weight.name].is_distributed = True
+    # set is_distributed for splited bias
```
Comment (wangxicoding) [inline python/paddle/distributed/collective.py:1016]: API文档可以补清楚一些，axis=0以及axis=1时的切法。注释也可以再描述清楚一点，为何column切分时bias也需要切分。

**Review by wangxicoding — DISMISSED** [review]
LGTM

File: python/paddle/distributed/collective.py (hunk 1 above)
Comment (JZ-LIANG) [reply python/paddle/distributed/collective.py:1016]: done~

**Review by wangxicoding — DISMISSED** [review]

**Review by sandyhouse — DISMISSED** [review]
LGTM

**Review by wangxicoding — APPROVED** [review]
LGTM

**Pull request merged**